From 45c529405a747e7be7cc229913393fa34f5cd4ff Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Thu, 14 Nov 2019 09:45:44 +0100
Subject: [PATCH 02/14] Do not set NM_CONTROLLED=no

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
RH-MergeRequest: 9: Synchronize RHEL 9 changes to RHEL 10
RH-Jira: RHEL-40107 RHEL-40679
RH-Acked-by: Ani Sinha <anisinha@redhat.com>
RH-Commit: [2/14] 3873d86dfc8a13e160b82cbdf3e9aaa52eedb37f (mrezanin/centos-git-hyperv-daemons)

patch_name: 0002-Do-not-set-NM_CONTROLLED-no.patch
present_in_specfile: true
location_in_specfile: 2
---
 hv_set_ifconfig.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/hv_set_ifconfig.sh b/hv_set_ifconfig.sh
index d10fe35..3dd064c 100644
--- a/hv_set_ifconfig.sh
+++ b/hv_set_ifconfig.sh
@@ -51,7 +51,6 @@
 
 
 echo "IPV6INIT=yes" >> $1
-echo "NM_CONTROLLED=no" >> $1
 echo "PEERDNS=yes" >> $1
 echo "ONBOOT=yes" >> $1
 
-- 
2.39.3

